src/fides/api/ops/schemas/shared_schemas.py

```diff
@@ -1,24 +0,0 @@
-from typing import Optional
-
-from fideslang.validation import FidesKey
-
-
-class FidesOpsKey(FidesKey):
-    """
-    Overrides fideslang FidesKey validation to throw ValueError
-    """
-
-    @classmethod
-    def validate(cls, value: Optional[str]) -> Optional[str]:
-        """Throws ValueError if val is not a valid FidesKey"""
-        if value == "<instance_fides_key>":
-            # Ignore <instance_fides_key> in saas templates.  This value will be replaced with a
-            # user-specified value.
-            return value
-
-        if value is not None and not cls.regex.match(value):
-            raise ValueError(
-                "FidesKey must only contain alphanumeric characters, '.', '_' or '-'."
-            )
-
-        return value
```

Comment:
Getting rid of FidesOpsKey in favor of an updated FidesKey.  

I've updated Fides Key to allow ">" "<" characters pending approval.  Also the error that FidesKey throws has been updated to be a subclass of a ValueError so can be converted into a pydantic ValidationError. 


The only behavior I didn't move to FidesKey was allowing the value argument to be Optional in FidesKey.value.  I think on our end we can just use Optional[FidesKey] for this.